docs/source/howtos/add_version.rst

```diff
@@ -0,0 +1,106 @@
+How To Add a Version To an Instrument
+=====================================
+
+ResINS strives to model all :term:`versions<version>` of any given :term:`INS`
+:term:`instrument` - old, new, and planned - but it is inevitable that some will
+not be natively supported *yet*. Fortunately, while not as straightforward as
+:doc:`creating a new instrument<add_instrument>`, ResINS aims to make adding
+new :term:`versions<version>` as simple as possible:
+
+.. contents::
+    :backlinks: entry
+    :depth: 2
+    :local:
+
+
+How to add a version for personal use
+-------------------------------------
+
+Adding a :term:`version` to an :term:`instrument` only locally (without
+:ref:`submitting to ResINS<howto-version-resins>`) is the more difficult task
+since ResINS does not have an extension/add-ons mechanism. Therefore, to create
+a version, it is necessary to resort to either creating an entire instrument or
+working with Python dictionaries instead of YAML files:
+
+.. _howto-version-yaml:
+
+Using YAML files
+^^^^^^^^^^^^^^^^
+
+It is currently impossible to simply append a new :term:`version` to an existing
```

Comment:
```suggestion
It is currently impossible to dynamically append a new :term:`version` to an existing
```

I initially read this as "new instrument versions don't go into the same YAML file" which is incorrect and not what this is referring to.